core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -70,6 +65,29 @@
 
   private final AtomicBoolean keepRunning = new AtomicBoolean(true);
 
+  public enum NodeData {
```

Comment:
This enum contains members whose names correspond to the attributes of an instance of a Fate transaction. When storing or accessing these attributes you have to supply the Fate transaction id. I would be more inclined to support a name like `TransactionData`, `TxData`, or something like that. I think the word "node" is coming from the fact that the data is stored in a ZK node. The name isn't really descriptive to someone looking at this code that doesn't know the inner workings of the persistence.

Reply:
`TxData` is nice and short and provides context and may be an improvement over node